README.md

```diff
@@ -26,8 +26,10 @@ An extension to [PWA](https://wordpress.org/plugins/pwa/) to enable `add to home
 - Offline reading by caching assets and pages.
 
 ### Offline Form Submission.
-- Submit gravity form even if you are offline. Plugin will send form response once you get back online.
-- Add form routes to enable offline form submission in plugin settings page.
+- Submit gravity form even if you are offline. The plugin will send form response once you get back online.
+- The offline form submission will only work for the routes added to the plugin settings page.
+- If no routes are set the offline form submission script will not be enqueued.
+- Add form routes to enable offline form submission in the plugin settings page.
```

Comment:
This line is pointing the same thing as line no 30
`The offline form submission will only work for the routes added to the plugin settings page.`

You can remove this line.